GPU/Vulkan/FramebufferVulkan.cpp

```diff
@@ -265,8 +265,14 @@ void FramebufferManagerVulkan::ReformatFramebufferFrom(VirtualFramebuffer *vfb,
 	// to exactly reproduce in 4444 and 8888 formats.
 
 	if (old == GE_FORMAT_565) {
-		// Previously started a render pass here to clear, but better to just, well, explicitly clear.
-		draw_->Clear(Draw::FBChannel::FB_COLOR_BIT | Draw::FBChannel::FB_STENCIL_BIT, 0, 0.0f, 0);
+		// We have to bind here instead of clear, since it can be that no framebuffer is bound.
+		// The backend can sometimes directly optimize it to a clear.
+		draw_->BindFramebufferAsRenderTarget(vfb->fbo, { Draw::RPAction::CLEAR, Draw::RPAction::KEEP, Draw::RPAction::CLEAR }, "ReformatFramebuffer"); 
```

Comment:
It was reported somewhere that there can be bugs when depthAction != stencilAction in some drivers.  Not sure how worried to be.

-[Unknown]

Reply:
Right, but we might want to take care of that in the RenderManager instead in that case... Hm.